src/api/activity/notifications.rs

```diff
@@ -0,0 +1,216 @@
+//! Github Notifications API
+
+use crate::models::activity::Notification;
+use crate::models::activity::ThreadSubscription;
+use crate::Octocrab;
+use crate::Page;
+
+type DateTime = chrono::DateTime<chrono::Utc>;
+
+/// Handler for GitHub's notifications API.
+///
+/// Created with [`ActivityHandler::notifications`].
+///
+/// [`ActivityHandler::notifications`]: ../struct.ActivityHandler.html#method.notifications
+pub struct NotificationsHandler<'octo> {
+    crab: &'octo Octocrab,
+}
+
+impl<'octo> NotificationsHandler<'octo> {
+    pub(crate) fn new(crab: &'octo Octocrab) -> Self {
+        Self { crab }
+    }
+
+    /// Gets a notification by their id.
+    pub async fn get(&self, id: impl Into<u64>) -> crate::Result<Notification> {
```

Comment:
Would you be able to provide `no_run` examples in documentation for each of these methods like the other modules? It will at least test that it compiles.